unison-src/new-runtime-transcripts/utf8.output.md

```diff
@@ -90,12 +90,12 @@ test> checkRoundTrip greek
     
       checkRoundTrip : Text -> [Result]
       greek          : Text
-    test.kqfpde2g5a  (Unison bug, unknown term)
+      greekTest      : [Result]
   
```

Comment:
What's the story behind the bug this no longer displays, again?

Reply:
There's some bug with anonymous test watches, which is a little known feature where you can get Unison to make up a GUID name for a test. But since you generally want your tests to have names, we've never gotten around to fixing. I think there's a ticket for it somewhere...